.gitignore

```diff
@@ -1 +1,2 @@
 node_modules
+test.md
```

Comment:
- [ ] You did well 👌at implementing the requirements of the project💯, but I see that you put the ``` test.md ``` file in the gitignore, but you didn't add the file to the directory, so please😊 put the test.md file in the directory of the project and put this sentence which inside the requirements.